templates/helpers/jsesc.js

```diff
@@ -0,0 +1,5 @@
+var jsesc = require('jsesc');
+
+module.exports = (value) => jsesc(
+    value.replace(/[\r\n]{1,2}/g, '')
```

Comment:
You're correct, problem is that i thought jsesc was doing weird escaping but it was handlebars. I will push new commits later today.